src/App.js

```diff
@@ -27,20 +30,23 @@ function App() {
   }, [pathname]);
 
   return (
-    <Routes>
-      <Route path="/" element={<UserRoutes />}>
-        <Route path="home" element={<Home />} />
-        <Route path="profile" element={<Profile />} />
-        <Route path="explore" element={<Explore />} />
-        <Route path="post/:postID" element={<PostPage />} />
-        <Route path="newpost" element={<NewPost />} />
-      </Route>
-      <Route element={<AuthRoutes />}>
-        <Route path="/login" element={<Login />} />
-        <Route path="/signup" element={<Signup />} />
-      </Route>
-      <Route path="*" element={<Page404 />} />
-    </Routes>
+    <>
+      <Routes>
+        <Route path="/" element={<UserRoutes />}>
+          <Route path="home" element={<Home />} />
+          <Route path="profile" element={<Profile />} />
+          <Route path="explore" element={<Explore />} />
+          <Route path="post/:postID" element={<PostPage />} />
+          <Route path="newpost" element={<NewPost />} />
+        </Route>
```

Comment:
Whats the requirement of new route for new post?

Reply:
for creating a new post